grails-app/domain/org/pih/warehouse/invoice/InvoiceItemType.groovy

```diff
@@ -0,0 +1,18 @@
+package org.pih.warehouse.invoice
+
+enum InvoiceItemType {
+
+    INVERSE('Inverse'),
+    PREPAYMENT('Prepayment'),
+    REGULAR('Regular')
+
+    String type
+
+    String toString() {
+        return name()
```

Comment:
```suggestion
        return type
```
Either that or I'd just remove the type field entirely since it's not being used anywhere but ultimately it's not a big deal.

Reply:
@EWaterman i replaced InvoiceItemType with inverse boolean